connector/src/main/java/org/jboss/as/connector/subsystems/jca/Constants.java

```diff
@@ -50,7 +50,7 @@ public class Constants {
     static final String TX = "TX";
     static final String NON_TX = "NonTX";
 
-    static final Boolean ELYTRON_MANAGED_SECURITY = Boolean.FALSE;
+    static final Boolean ELYTRON_MANAGED_SECURITY = Boolean.TRUE;
```

Comment:
This requires a management API and xsd version bump. (XSD because the current one documents the default behavior as false.)

It needs a transformer that specifically sets the undefined value to true, so the legacy servers run with the settings the domain config indicates.

The parsers for legacy xsd namespaces should also set 'false' if the value is not configured.